ros2topic/test/test_echo_pub.py

```diff
@@ -247,9 +247,15 @@ def publish_message():
                             'Echo CLI did not print expected message'
                         )
                     else:
-                        assert not command.output, (
-                            'Echo CLI should not have received anything with incompatible QoS'
-                        )
+                        if 'rmw_fastrtps' in get_rmw_implementation_identifier():
```

Comment:
@mm318 mind to add a TODO for removal once https://github.com/ros2/rmw_fastrtps/issues/356 is addressed?

Reply:
Done.